conversion.md

```diff
@@ -0,0 +1,111 @@
+# Conversion to OCI Runtime Configuration
+
+When extracting an OCI Image into an [OCI Runtime bundle][oci-runtime-bundle], two orthogonal components of the extraction are relevant:
+
+1. Extraction of the root filesystem from the set of [filesystem layers](layers.md).
```

Comment:
In the same way that we would have to test runtime implementations that similarly make the bundle or config opaque to the user. This document only concerns tools that implement this conversion (that intend to actually also implement either a runtime or to produce a `config.json` to the user).